### ISSUE TITLE
🐛 BUG: workerd cpp errors when reloading

### ISSUE THREAD

**Issue by Cherry** [cloudflare/workerd#1366]
### Which Cloudflare product(s) does this pertain to?

Wrangler core

### What version(s) of the tool(s) are you using?

3.13.0

### What version of Node are you using?

18.18.0

### What operating system are you using?

Windows 10

### Describe the Bug

- npm create cloudflare@latest
- npm run start
- Make a change to the worker, and you'll see an error like this:

```
⎔ Starting local server...
[wrangler:inf] Ready on http://0.0.0.0:8787
[wrangler:inf] - http://192.168.50.240:8787
[wrangler:inf] - http://127.0.0.1:8787
[wrangler:inf] - http://172.24.32.1:8787
kj/async-io-win32.c++:982: warning: Bind address resolved to multiple addresses.  Only the first address will be used.  If this is incorrect, specify the address numerically.  This may be fixed in the future.; addrs[0].toString() = 127.0.0.1:49849
[wrangler:inf] GET /cdn-cgi/ProxyWorker/pause 204 No Content (0ms)
[wrangler:inf] GET /cdn-cgi/ProxyWorker/pause 204 No Content (18ms)
[wrangler:inf] GET /cdn-cgi/ProxyWorker/pause 204 No Content (1ms)
[wrangler:inf] GET /cdn-cgi/ProxyWorker/play 204 No Content (1ms)
[wrangler:inf] GET /cdn-cgi/ProxyWorker/pause 204 No Content (0ms)
⎔ Reloading local server...
[wrangler:inf] GET /cdn-cgi/ProxyWorker/pause 204 No Content (0ms)
workerd/jsg/util.c++:275: error: e = kj/async-io-win32.c++:385: failed: WSARecv(): cloudflare/workers-sdk#64 The specified network name is no longer available.
stack: 7ff6c7925100 0 0 7ff6c7932cba 0 0 7ff6c7f0e6a8; sentryErrorContext = jsgInternalError
✘ [ERROR] { message: 'Uncaught Error: internal error', cause: {} }


workerd/jsg/util.c++:275: error: e = kj/async-io-win32.c++:385: failed: WSARecv(): cloudflare/workers-sdk#64 The specified network name is no longer available.
stack: 7ff6c7925100 0 0 7ff6c7932cba 0 0 7ff6c7f0e6a8 7ff6c7932c6c 7ff6c7f086fc 7ff6c8a8b369 7ff6c8b8fff0 7ff6c8ab34be 7ff6c8a876da 7ff6c6bfee49 7ff6c6bff7e9 7ff6c6bffa52 7ff6c6c157ef 7ff6c6c155e1 7ff6c7cf3cec 7ff6c7f06525 7ff6c7f0735e 7ff6c78e23f5 7ff6c78e8415 7ff6c78e9558 7ff6c78e0807 7ff6c69dae72 7ff6c69de8c9 7ff6c69de762 7ff6c69de74c 7ff6c7956d8c 7ff6c795d816 7ff6c7957321; sentryErrorContext = jsgInternalError
workerd/io/worker.c++:1790: info: uncaught exception; source = Uncaught (in promise); exception = Error: internal error
workerd/io/io-context.c++:394: info: uncaught exception; exception = workerd/jsg/_virtual_includes/jsg\workerd/jsg/value.h:1334: failed: jsg.Error: internal error
kj/async-io-win32.c++:982: warning: Bind address resolved to multiple addresses.  Only the first address will be used.  If this is incorrect, specify the address numerically.  This may be fixed in the future.; addrs[0].toString() = 127.0.0.1:49849
[wrangler:inf] GET /cdn-cgi/ProxyWorker/play 204 No Content (0ms)
```

It's possible this happens on Windows only, but does not occur with 3.12.0.

**Comment by admah** [comment]
@Cherry we have published a new version of Wrangler (3.13.1) that reverts the change that was causing issues. You should be able to update and develop your Worker as expected.

**Comment by RamIdeas** [comment]
Hi @Cherry. Thanks for reporting this issue. I was able to reproduce this on Windows. However, your worker will remain completely functional.

Specifics: the new InspectorProxyWorker has a websocket connection to the runtime inspector server of the previous version of your worker. This error message is spat out when that version of your worker is shutdown. The new version of your worker is spun up and the InspectorProxyWorker connects to that successfully.

I will follow up with the runtime team to silence this error message so will leave this issue open.

In the meantime, can you confirm my observation on your setup: your worker is still functional even after that error message?

**Comment by Cherry** [comment]
Thanks for the information. I can confirm the worker is still functional, yes - it's just a very loud and scary error 😆

**Comment by RamIdeas** [comment]
Thanks for confirming! We're going to move this issue into the workerd repo to keep track

TL;DR from above: `WebSocket` error events handled in javascript still logs extraneous error messaging from workerd

**Comment by Cherry** [comment]
This isn't an issue anymore, so going to close.
